src/Components/Server/src/Builder/RazorComponentsEndpointRouteBuilderExtensions.cs

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Server;
+using Microsoft.AspNetCore.Routing;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    /// <summary>
+    /// Extensions for <see cref="RazorComponentsEndpointBuilder"/>
+    /// </summary>
+    public static class RazorComponentsEndpointRouteBuilderExtensions
```

Comment:
```suggestion
    public static class ComponentEndpointRouteBuilderExtensions
```

Reply:
dibe

Reply:
> dibe

This is done but with a letter to the left on the keyboard #fatfingers